README.md

```diff
@@ -74,6 +74,38 @@ overlays <- list(create_overlay("bpmn_element_id_1", "42"), create_overlay("bpmn
 bpmnVisualization::display(bpmn_file, overlays)
 ```
 
+
+### Integrate in Shinny Applications
+
+The following displays a BPMN diagram provided as example by the package with an overlay on top of a BPMN element
+
+```r
+# Install and load packages
+install.packages("shiny")
+library(shiny)
+devtools::install_github("process-analytics/bpmn-visualization-R")
+library(bpmnVisualization)
+
+displayBpmn <- function() {
+    bpmn_file <- system.file("examples/Travel_Booking.bpmn", package = "bpmnVisualization")
+    overlays <- list(bpmnVisualization::create_overlay("_6-203", "9"))
+    bpmnVisualization::display(bpmn_file, overlays = overlays)
+}
+
+ui <- shinyUI(fluidPage(
+    titlePanel("Display bpmn diagrams with execution data"),
+    bpmnVisualizationOutput('bpmnContainer')
+  )
+)
+
+server = function(input, output) {
+    # renderBpmnVisualization is the R bridge function to the html widgets
+    output$bpmnContainer <- renderBpmnVisualization({ displayBpmn() })
+}
+
+shinyApp(ui = ui, server = server)
```

Comment:
```suggestion
shinyApp(ui, server)
```

Reply:
This line is taken from the shinny documentation, so I will leave it as it is.

Reply:
No need to put the variable name in R, like `ui =`, when the parameter are set in the defined order (in function) 😉 
http://adv-r.had.co.nz/Functions.html#function-arguments

I test without them, and it works. It avoid duplication.

Reply:
I know that, but thanks for the link.
My remark was about being consistent with the usual way of integrating shinny code.
Anyway, I don't want to loose time on this, so I am applying your suggestion.